game/scripts/npc/abilities/treant_living_armor.txt

```diff
@@ -35,18 +35,18 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "total_heal"                                      "60 120 180 240 480 960"
+        "total_heal"                                      "60 100 140 180 360 720"
```

Comment:
I calculated the old values for Living Armor total heal, it was: 60/120/180/240/720/1440.